.travis.yml

```diff
@@ -3,23 +3,27 @@ language: generic
 
 sudo: required
 
-script:
-  - ./build.sh
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+jobs:
+  include:
+    - script: ./build.sh
+    - script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+    - stage: GitHub Release
```

Comment:
![abhi](https://user-images.githubusercontent.com/15213246/28000988-ef42403a-6546-11e7-9991-202159b6eba1.png)
Deploy build only get started when the both the test cases successfully complete